docs/.eleventy.js

```diff
@@ -55,11 +55,23 @@ module.exports = function(eleventyConfig) {
     // Load site-specific data
     const siteName = process.env.ESLINT_SITE_NAME || "en";
 
+    /**
+     * Determines whether the given version is a prerelease.
+     * @returns {boolean} `true` if it is a prerelease, `false` otherwise.
+     */
+    async function isPreRelease() {
```

Comment:
Can we rename this to `isPrereleasePhase`? "isPreRelease" sounds like it should return `true` only if the current branch is a prerelease branch.

(the same for eleventy global data variable name)